dev/com.ibm.ws.jaxws.common/bnd.bnd

```diff
@@ -16,23 +16,31 @@ Bundle-Name: JAX-WS Common
 Bundle-SymbolicName: com.ibm.ws.jaxws.common
 Bundle-Description: IBM JAX-WS Common; version=${bVersion}
 
```

Comment:
Only a few classes are different between jaxws.common and jaxws.2.3.common.  It is odd to say it is a common bundle when it needs to be duplicated.  This would seem to be nicely refactored so that the common stuff is truly common and you could still have a 2.3.common, but it just have the bnd differences.  For classes that are not common, having them in their own bundle will be helpful.

Reply:
Will open an issue to address this.